packages/react/tests/LocalNavigation.test.tsx

```diff
@@ -27,24 +27,27 @@ describe('react: LocalNavigation', () => {
     }
   });
 
-  it('should handle click events', () => {
-    const onClick = jest.fn();
+  // we should not be testing for onClick explicitly as react-aria suggests that we need to use onPress
```

Comment:
The `LocalNavigationItem` is still allowing `onClick` in both its props and its typings. Simply disabling this test does not address the issue, and additionally will break the backwards compatibility as this is a shared common component library used throughout micro-webapps.

Reply:
@p44dyoachim Yes i didn't wanted cause breaking change that's why i didn't limited onClick. but ideally on consumer we should stop using onClick. disabling test will not break any functionality, disabling prop on component will.